source/developers/component_loading.markdown

```diff
@@ -9,17 +9,15 @@ sharing: true
 footer: true
 ---
 
-A component will be loaded on start if a section (ie. `light:`) for it exists in the config file. A component can also be loaded if another component is loaded that depends on it. When loading a component Home Assistant will check the following paths:
+A component will be loaded on start if a section (ie. `light:`) for it exists in the `configuration.yaml` file. A component can also be loaded if another component is loaded that depends on it. When loading a component Home Assistant will check the following paths:
 
- * `<config directory>/custom_components/<component name>`
+ * `.homeassistant/custom_components/<component name>`
  * `homeassistant/components/<component name>` (built-in components)
+    
+If Home Assistant is installed in an [`venv`](/docs/installation/virtualenv/) you find the components folder under the following subfolder: `lib/python3.5/site-packages/homeassistant/components/<component name>`
```

Comment:
Worth making the Python version a place holder, since some people will be running 3.6, and 3.7 will be out soon too.

Reply:
Like `3.x`?

Reply:
That and something like `replace 3.x with your Python version` just in case people aren't paying attention, and go looking for a directory literally called 3.x